src/app/legacy/containers/ConsentBanner/Banner/index.canonical.jsx

```diff
@@ -25,7 +25,7 @@ const AcceptButton = ({ message, onClick, dataAttribute }) => (
   </button>
 );
 
-const RejectButton = ({ message, href, onClick, dataAttribute }) => (
+export const RejectButton = ({ message, href, onClick, dataAttribute }) => (
```

Comment:
See https://github.com/bbc/simorgh/commit/1109ef8911d83bcb1d5b8c931afd591c0ee378a5 for why I had to do this